.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+name: Release workflow
+
+on:
+  release:
+    types: [published]
```

Comment:
This is not what we want. For this tool.

We don't really want to make releases on github for this, this is just overhead.

This should have a workflow_dispatch so we can trigger it manually every 2 years that we need to.

Reply:
This particular trigger also measn this job will run no matter what release is published, for whatever component that might be added here. And this kind of action-at-a-distance is undesirable.

Reply:
Sorry I had this enabled for a 30 min window to make a new release that I could use in the pipeline. 

This is no longer there.

Is the new change more what you were seeking?

Reply:
Yeah, for this tool which is kind of a one-off this is what we want.